dcos-log/journal/reader/read.go

```diff
@@ -8,6 +8,7 @@ import (
 	"time"
 
 	"github.com/coreos/go-systemd/sdjournal"
+	"github.com/Sirupsen/logrus"
```

Comment:
The new location for this library is at `github.com/sirupsen/logrus` -- probably want to change to that if we are able.